modules/tinymce/src/plugins/accordion/main/ts/api/Events.ts

```diff
@@ -0,0 +1,10 @@
+import Editor from 'tinymce/core/api/Editor';
+import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
+
+export const fireToggleAccordionEvent =
+  (editor: Editor, element: HTMLDetailsElement, state: boolean): EditorEvent<{ element: HTMLDetailsElement; state: boolean }> =>
```

Comment:
Make this a AccordionEvent type so that you can re-use them in tests etc.

Reply:
Exported the event types for the tests here.